src/components/TeamSection/TeamSection.tsx

```diff
@@ -1,39 +1,48 @@
-import React from "react";
+import React, { useCallback } from "react";
 import { useState } from "react";
 import Team from "./Team";
 import * as S from "./TeamSection.styles";
-import { executiveTeam, projectManagers } from "./TeamInformation";
+import { executiveTeam, projectManagers, alumniTeam } from "./TeamInformation";
+import { TeamCategory } from "./Team";
 
 type TeamSectionProps = {
   desktopView: boolean;
 };
 
 const TeamSection = (props: TeamSectionProps) => {
-  const [toggle, setToggled] = useState(false);
+  const [selectedCategory, setSelectedCategory] = useState<TeamCategory>("executives");
 
-  const getTeamMembers = () => {
-    return toggle ? projectManagers : executiveTeam;
-  };
+  const getTeamMembers = useCallback(() => {
```

Comment:
this should be a memoized value, since it doesn't change unless `selectedCategory` changes